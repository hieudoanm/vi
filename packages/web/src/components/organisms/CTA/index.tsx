import styled from '@emotion/styled';
import Button from '@mui/material/Button';
import Link from 'next/link';
import React from 'react';

const StyledSection = styled.section`
  clip-path: polygon(0 0, 100% 14%, 100% 100%, 0% 100%);
`;

const CTA: React.FC = () => {
  return (
    <StyledSection className="bg-cyan-50 py-20">
      <div className="flex flex-col justify-center items-center pt-20 pb-10">
        <h1 className="text-center mx-auto mb-8 text-4xl font-bold">
          Be the first to get the beta
        </h1>
        <Link href="/api/auth/login" passHref>
          <Button type="button" variant="contained" size="large">
            Get early access
          </Button>
        </Link>
        <p className="text-sm font-light pt-4">No credit card required.</p>
      </div>
    </StyledSection>
  );
};

export default CTA;
