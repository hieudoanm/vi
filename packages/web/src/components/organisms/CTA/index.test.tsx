import { render } from '@testing-library/react';
import CTA from '.';

describe('CTA', () => {
  test('render default', () => {
    const wrapper = render(<CTA />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
