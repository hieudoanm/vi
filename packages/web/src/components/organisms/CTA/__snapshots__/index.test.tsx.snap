// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTA render default 1`] = `
<div>
  <section
    class="bg-cyan-50 py-20 css-9ptmzd"
  >
    <div
      class="flex flex-col justify-center items-center pt-20 pb-10"
    >
      <h1
        class="text-center mx-auto mb-8 text-4xl font-bold"
      >
        Be the first to get the beta
      </h1>
      <a
        class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButtonBase-root  css-jh47zj-MuiButtonBase-root-MuiButton-root"
        href="/api/auth/login"
        tabindex="0"
        type="button"
      >
        Get early access
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
      <p
        class="text-sm font-light pt-4"
      >
        No credit card required.
      </p>
    </div>
  </section>
</div>
`;
